lib/str.c

```diff
@@ -1514,6 +1514,12 @@ grn_str_len(grn_ctx *ctx,
   return len;
 }
 
+int
```

Comment:
```suggestion
bool
```

Functions named `isXXX()` should return `true` or `false`.

Reply:
fix: [95ffcdb](https://github.com/groonga/groonga/pull/2331/commits/95ffcdba72041e7e6b3b71c72888f70d9e3d15d2) Sure!